common/persistence/class.SqlKvDriver.php

```diff
@@ -76,30 +80,31 @@ public function set($id, $value, $ttl = null, $nx = false)
         try {
             $expire = is_null($ttl) ? 0 : time() + $ttl;
 
-            // we need int to have safe incr and decr methods
-            $encoded = is_int($value) ? $value : base64_encode($value);
-            $platformName = $this->sqlPeristence->getPlatForm()->getName();
+            $encoded = base64_encode($value);
```

Comment:
We are loosing backward compatibility at that moment, right? Was that on purpose?

Reply:
the purpose here is to unify value in KV, but agreed Im fixing backward compatibility

Reply:
Unification is awesome, but that we need possibility to restore existing values properly, encoded in the old-fashion

Reply:
I'm keeping this behaviour but at decoding, Im taking care of old fashion way